pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -262,17 +270,94 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	ghs.logger.Sugar().Debugf("current date: %v", currentDate)
 
 	ghs.logger.Sugar().Debug("creating a new github client")
-	client := github.NewClient(ghs.client)
 
-	repos, _, err := client.Repositories.List(ctx, ghs.cfg.GitHubOrg, nil)
+	graphqlClient := githubv4.NewClient(ghs.client)
+
+	variables := map[string]interface{}{
+		"login": githubv4.String(ghs.cfg.GitHubOrg),
+	}
+
+	// we need to check if the provided org in config.yml is a user or an organization
+	var login_query struct {
+		Organization struct {
+			Login githubv4.String
+		} `graphql:"organization(login: $login)"`
+	}
+
+	var provided_org_is_org bool
+
+	err := graphqlClient.Query(context.Background(), &login_query, variables)
 	if err != nil {
-		ghs.logger.Sugar().Errorf("Error getting repos", zap.Error(err))
+		ghs.logger.Sugar().Info("Org provided not found or org provided is user", zap.Error(err))
```

Comment:
This should be an error

Reply:
I guess... it should be an error it what they passed through in the config.yml for org is **neither** a user or org that can be found.

If they pass a user, then this is just info stating a user was passed instead of an org.


So I could do 2 look ups (one for org and if it doesn't exist, do a look up for user). If both of those look ups fail, then it should result in an error.

Reply:
follow up issue: https://github.com/liatrio/liatrio-otel-collector/issues/51